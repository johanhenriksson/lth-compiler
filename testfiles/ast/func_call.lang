int main() {
    main();
    main(5);
    main(5,6);
}
