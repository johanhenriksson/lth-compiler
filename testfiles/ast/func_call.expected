Program
  List
    Function
      TypeId
      IdDecl
      List
      Block
        List
          CallStmt
            FuncExpr
              IdUse
              List
          CallStmt
            FuncExpr
              IdUse
              List
                IntLiteral
          CallStmt
            FuncExpr
              IdUse
              List
                IntLiteral
                IntLiteral
